src/Pirouette/Symbolic/Prover/Runner.hs

```diff
@@ -21,37 +22,69 @@ import qualified Test.Tasty.HUnit as Test
 data AssumeProve lang = Term lang :==>: Term lang
   deriving (Eq, Show)
 
-type IncorrectnessResult lang = Maybe (Path lang (EvaluationWitness lang))
-
 -- | Parameters for an incorrectness logic run
 data IncorrectnessParams lang = IncorrectnessParams
-  { ipDefinitions :: Program lang,
-    ipTarget :: Term lang,
-    ipCondition :: AssumeProve lang,
-    ipMaxCstrs :: Int
+  { ipTarget :: Term lang,
+    ipTargetType :: Type lang,
+    ipCondition :: AssumeProve lang
   }
 
+-- | The result type of 'runIncorrectnessLogic', returning the first (if any)
+--  path that is a counterexample to the supplied 'IncorrectnessParams'.
+--  If you would like finer grained control, look at 'execIncorrectnessLogic'.
+type IncorrectnessResult lang = Maybe (Path lang (EvaluationWitness lang))
+
+runIncorrectnessLogic1 ::
```

Comment:
What's `1` here? At first I thought it's the number of counterexamples to return, but now I'm not sure what's the difference from the un-`1`-ed version.

Reply:
The one is that it uses a single term, it doesn't take in a set of definitions (`PrtUnorderedDefs`). I'll see if I come up with a better name

Reply:
Renamed to `runIncorrectnessLogicSingl`